packages/buidler-core/test/internal/buidler-evm/stack-traces/test.ts

```diff
@@ -585,7 +625,22 @@ async function runCallTransactionTest(
   return trace as CallMessageTrace;
 }
 
+const solidity05Compilers = [
```

Comment:
I think everything would be simpler if these were `CompilerOptions` objects, especially `defineDirTests`.